TAF/BDT/Steps/SetupSteps.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NUnit.Framework;
+using TAF.Core.Browser;
+using TAF.Core.Utilities;
+using TAF.Helper;
+using TAF.Utilities.Logger;
+using TAF.Web.Pages;
+
+namespace TAF.Tests.BDT.Steps
+{
+    [Binding]
+    public class SetupSteps
+    {
+        private MainPage _mainPage;
+
+        [BeforeScenarioBlock]
+        public void LoggerSetup() => Logger.NewLogger(TestContext.CurrentContext.Test.Name, TestContext.CurrentContext.TestDirectory);
+
+        [Given(@"navigation to epam\.com page")]
+        public void GivenNavigationToEpam_ComPage()
+        {
+            _mainPage = new();
+            Logger.Info("Start new test");
+            Browser.NewBrowser.GoToUrl(TestSettings.ApplicationUrl);
+            Waiters.WaitForPageLoad();
+        }
+
+        [Given(@"accept all cookies")]
+        public void GivenAcceptAllCookies()
```

Comment:
Please review that it is better to left only GENERAL steps in setup class as browser and logger setup.
Steps that are connected with pages should be placed in appropriate classes. Imagine that we need to create a test that will check text on Cookie banner. So step that accept cookie banner is not for setup anymore